docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
       DJANGO_SETTINGS_MODULE: config.settings.local
       REDIS_URL: redis://redis:6379
     volumes:
-      - .:/app
+      - .:/app:cached
```

Comment:
@wlonk I'm not sure I fully understand these caching settings (https://docs.docker.com/docker-for-mac/osxfs-caching/), but I read that this is a way to _significantly_ improve Docker performance on OS X. See https://github.com/docker/for-mac/issues/3499#issuecomment-538681270

Reply:
Read through those docs, and looks good. Let's try it out and keep an eye out for any problems.